x-pack/plugins/dataset_quality/common/api_types.ts

```diff
@@ -70,3 +76,5 @@ export const getDataStreamsDegradedDocsStatsResponseRt = rt.exact(
     degradedDocs: rt.array(degradedDocsRt),
   })
 );
+
+export const getDataStreamsDetailsResponseRt = rt.exact(dataStreamDetailsRt);
```

Comment:
Are we using this type? couldn't find the reference

Reply:
It's used [here](https://github.com/mohamedhamed-ahmed/kibana/blob/b68e2592d8fbe14ef09cf899568a4e6a6e4a3521/x-pack/plugins/dataset_quality/public/services/data_streams_stats/data_streams_stats_client.ts#L97) to validate server response.